CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Possible sections are:
 <!-- next-header -->
 ## [Unreleased] - ReleaseDate
 
+### Fixed
+
+- fix spelling in bbox
+
```

Comment:
Need a rebase (conflict with your other PR I just merged), then we're good to go 🙂